store.html

```diff
@@ -0,0 +1,143 @@
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="store.html">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="store-section">
+        <form class="form-container">
+          <p>Order your Karma wifi device today!</p>
+
+          <fieldset class="name">
+            <div class="name-group">
+              <label for="firstname">First name *</label>
+              <input id="firstname" type="text" name="username" />
+            </div>
+            <div class="name-group">
+              <label for="lastname">Last name *</label>
+              <input id="lastname" type="text" name="lastname" />
+            </div>
+          </fieldset>
+
+          <fieldset class="address">
+            <div class="address1">
+              <label for="address">Address *</label>
+              <input id="address" type="text" name="address" />
+            </div>
+            <div class="address2">
+              <label for="address2">Address 2</label>
+              <input id="address2" type="text" name="address2" />
+            </div>
+          </fieldset>
+
+          <fieldset class="city">
+            <div class="city-group">
+              <label for="city">City *</label>
+              <select id="city" name="city">
+                <option id="city-placeholder" value="city-placeholder" selected>
+                  Select your city...
+                </option>
+                <option value="london">London</option>
+                <option value="istanbul">Istanbul</option>
+                <option value="paris">Paris</option>
+              </select>
+            </div>
+            <div class="city-group">
+              <label for="postcode">Postcode *</label>
+              <input id="postcode" type="text" name="postcode" />
+            </div>
+          </fieldset>
+
+          <fieldset class="select-color">
+            <legend>Select a colour</legend>
+            <input
+              id="karmaorange"
+              type="radio"
+              name="color"
+              value="karmaorange"
+            />
+            <label for="karmaorange">Karma orange</label>
+            <input id="spacegrey" type="radio" name="color" value="spacegrey" />
+            <label for="spacegrey">Space Grey</label>
+          </fieldset>
+
+          <fieldset class="checkbox">
+            <input
+              id="terms-and-conditions"
+              type="checkbox"
+              name="terms-and-conditions"
+            />
+
+            <label for="terms-and-conditions"
+              >By placing your order you agree to Karma's
+              <span style="color: #e0633a"
+                ><a href="#">Terms and conditions.</a></span
+              >
```

Comment:
Unusual formatting - it breaks "visual flow" of the HTML document. I'd suggest to keep the closing bracket next to the element's name